lib/ExpensiMark.js

```diff
@@ -267,7 +267,7 @@ export default class ExpensiMark {
             },
             {
                 name: 'anchor',
-                regex: /<(a)[^><]*href\s*=\s*(['"])(.*?)\2(?:".*?"|'.*?'|[^'"><])*>(.*?)<\/\1>(?![^<]*(<\/pre>|<\/code>))/gi,
+                regex: /<(a)[^><]*href\s*=\s*(['"])(.*?)\2(?:".*?"|'.*?'|[^'"><])*>(.*?)<\/\1>(?![^<]*(<\/pre>|<\/code>))/gis,
```

Comment:
We generally also add Tests for all the markdown changes. Please check previous PRs or other Tests for reference.

Reply:
Sorry about my confusion